ai-docs/emoji-commit-ref.yaml

```
@@ -190,3 +190,85 @@
+        example: 'Closes #42'
+        description: 'References issue in the current repository'
+      
+      - pattern: '<org>/<repo>#<issue-number>'
+        example: 'Fixes user-org/repo#456'
+        description: 'References issue in another repository'
+    
+    linear_issues:
+      - pattern: '<team-key>-<issue-number>'
+        example: 'Fixes ENG-123'
+        description: 'References Linear issue using team key format'
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)